src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html

```diff
@@ -83,16 +82,29 @@
     <div class="panel margin-left margin-left-sm margin-right margin-right-sm">
         <div class="padding padding-lg margin-top margin-top-sm">
             <div class="row margin-top">
-                <div class="properties-container stats-container flex-horizontal padding padding-sm" data-label="Stats">
+                <div class="properties-container stats-container flex-horizontal padding padding-sm" data-label="General">
                     <div class="property">
-                        <h5><i class="icon-running-tasks"></i><span>Requests</span></h5>
-                        <strong data-bind="text: stats.count" class="stats-property"></strong>
+                        <h5><i class="icon-client"></i><span>Clients</span></h5>
```

Comment:
I wonder if label `Clients` is not ambiguous here. Let's suppose I have one application hosted on 3 nodes and it uses same certificate (app is load balanced) - then I'd say we have 1 client (I distinct certificate is used). 

Thoughts?

Reply:
'**Distinct IPs**' ? 
or
**Source IPs**  ?